### ISSUE TITLE
Disabling input devices on Hyprland configuration

### ISSUE THREAD

**Issue by leonardohn** [hyprwm/Hyprland#833]
I've been using a DualSense controller (PS5), and it has a touchpad that can work as a general input device on Linux. However, having the touchpad enabled at a window manager level makes many games to fail using its functionalities, so I would like to force Hyprland to not use it while at the same time not disabling it on a udev level so the games can still use it. Is it possible to do it in Hyprland current state?

**Comment by vaxerski** [comment]
not possible atm, planned tho.

**Comment by rscottyy** [comment]
I am also dealing with this problem.

**Comment by vaxerski** [comment]
done in 28c81fc71ec2593cdac6cbf945369b201c1e3e71

per-device config `enabled`, applies only to cursor devices (mice and touchpads)

**Comment by leonardohn** [comment]
I've been testing with this change for a few days and noticed that sometimes it doesn't work. This is how I configured the controller for both the wired and the wireless device names, respectively:
```
device:Sony Interactive Entertainment Wireless Controller Touchpad {
    enabled=no
}
device:Wireless Controller Touchpad {
    enabled=no
}
```
The mousepad only stops moving the cursor if I run `hyprctl reload` after plugging the controller.

**Comment by rscottyy** [comment]
I also have similar issues.

**Comment by vaxerski** [comment]
oops, fixed in b51222c0044a04869f0486544606e4f65df9995c

**Comment by TomCC7** [comment]
Hi, just wondering is there a similar option for keyboard devices? Found a related discussion [here](https://github.com/hyprwm/Hyprland/discussions/4283) but it's not answered.
